tests/wpt/web-platform-tests/css/cssom/null-owner.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<head>
+  <meta charset="utf-8">
+  <title>ownerNode null test</title>
+  <link rel="help" href="https://drafts.csswg.org/cssom/#dom-stylesheet-ownernode">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <style></style>
+</head>
+<body>
+  <script>
+  	test(function() {
+    let styleElement = document.querySelector("style").sheet;
+    assert_equals(styleElement.sheet.ownerNode, styleElement);
+    sheet.ownerNode.remove();
+    assert_equals(sheet.ownerNode,null);
```

Comment:
Oops, I just found this test must fail? because we don't actually have the `sheet` 👀

Reply:
yes, this test would fail. It was just a try to get advice from you. What should I write to check the ownerNode is style Element, The statement I wrote does not work.

Reply:
I think we can check it with `instanceof HTMLStyleElement`.

Reply:
This is wrong, `sheet` is not defined anywhere. You should do:

```js
let styleElement = document.querySelector("style");
let sheet = styleElement.sheet;
```